src/PendingObjects/UsesCall.php

```diff
@@ -12,6 +13,20 @@
  */
 final class UsesCall
 {
+    /**
+     * Contains a global before each hook closure to be executed.
+     *
+     * Array indices here matter. They are mapped as follows:
+     *
+     * - `0` => `beforeAll`
+     * - `1` => `beforeEach`
+     * - `2` => `afterEach`
+     * - `3` => `afterAll`
+     *
+     * @var array<int, Closure>
+     */
+    private $hooks = [];
```

Comment:
Thoughts on constants to reference these index values? I saw that plain indices are used in the `TestRepository` for the `TestRepository::use`  method, so I just stuck with that for now.